CHANGELOG.md

```diff
@@ -9,14 +9,20 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ### Added
 
+### Fixed
+
+### Changed
+
+## 0.3.0 - 2024-12-27
+
```

Comment:
Please, add an overview entry for the release.

Reply:
Sorry for my inattention, added.